srv/components/system/teardown.sls

```diff
@@ -63,6 +63,14 @@ Remove /var/log/journal:
   file.absent:
     - name: /var/log/journal
 
+Remove /var/log/crash:
```

Comment:
@ypise `crash` folder is created as part of our system install. Should it be retained during complete teardown, since this contains all components coredumps, or removal is the right step?

Reply:
The folder should be removed in case of complete teardown.
If required the mentioned packages should be captured using supportbundle command prior to teardown.